### PR TITLE
Erdikose/feature/save draggable ratio

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

* Saves draggable ratio and pane states to electron store. So configurations will be saved on start.

## Fixes

*

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/atoms/SplitView/SplitView.tsx [hunk 1]
```diff
@@ -158,6 +178,13 @@ const SplitView: FunctionComponent<SplitViewProps> = ({
     setDraggingLeftNav(false);
     setDraggingNavEdit(false);
     setDraggingEditRight(false);
+    dispatch(updateLeftWidth(leftWidth));
```
Comment (olensmar) [inline src/components/atoms/SplitView/SplitView.tsx:181]: perhaps just have one updatePaneConfiguration reducer method that takes an object with all values!?

**Review by olensmar — COMMENTED** [review]
LGTM but let's await @kambydyne also!

**Comment by olensmar** [comment]
@erdkse is this ok to review or do you have outstanding fixes?

**Comment by erdkse** [comment]
> @erdkse is this ok to review or do you have outstanding fixes?

Steen wanted me to do a fix. All panes stand evenly after toggling one of the panes. I will change it. My work on the system menu is almost done. I can complete this today.

**Review by olensmar — APPROVED** [review]
LGTM

**Pull request merged**